joomla/joomla-cms
# new feature: plugin modal view needed for quicker edit

in admin just the way joomla has module modal edit view, we need to have plugin edit view in modal to save its settings. we dont want to forward users to find and edit things
## Test Instructions:

i have showed in com_redirect with full functional language, helper and view files. so if you visit com_redirect
administrator/index.php?option=com_redirect
you will see how its working.

![image](https://cloud.githubusercontent.com/assets/1936565/8352947/1f004782-1b5b-11e5-8e9b-9c5ce3f7530b.png)
thanks
